pkg/controller/metrics/provision_underway_collector.go

```diff
@@ -30,6 +30,10 @@ var (
 type provisioningUnderwayCollector struct {
 	client client.Client
 
+	// minDuration is the minimum duration afer which clusters provisioning
+	// will start reporting the metric. Only used when non-zero.
```

Comment:
"Only used when non-zero", could you clarify that if left zero the metric is always reported? When first read I thought it meant we would never report it if it was zero.

Reply:
updated